e2e/features/steps/registration.steps.ts

```diff
@@ -0,0 +1,39 @@
+import { Given, When, Then, BeforeAll, AfterAll } from "@cucumber/cucumber";
+import { expect, page, setup, teardown } from "../../utils/index";
+import { baseUrl } from "../../utils/consts";
+
+BeforeAll(setup);
+
+AfterAll(teardown);
+
+Given("the user is on the registration page", async function () {
+  await page.goto(baseUrl + "/register");
+});
+
+When(
+  "the user enters an invalid {string} address \\(e.g., missing @)",
+  async function (emailAddress: string) {
+    await page.getByLabel("Email").fill(emailAddress);
+  }
+);
+
+When("the user enters a name and a password", async function () {
+  await page.getByLabel("Név").fill("InvalidEmailTest");
+  await page.getByLabel("Jelszó").fill("!nval1dEm41ilTest");
+});
+
+When("the user clicks the {string} button", async function (register: string) {
+  await page.getByLabel(register).click();
+});
+
+Then(
+  "the system should display an error message indicating the email address is invalid",
+  async function () {
+    let alert = page.getByRole("alertdialog");
```

Comment:
```suggestion
    const alert = page.getByRole("alertdialog");
```

Might be const if no redeclaration.